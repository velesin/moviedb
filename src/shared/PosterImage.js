import React from 'react';

function fullUrl(path) {
    return `https://image.tmdb.org/t/p/w90_and_h134_bestv2/${path}`;
}

export function PosterImage({ path, title }) {
    return (
        <img src={ fullUrl(path) } alt={ title } />
    );
}