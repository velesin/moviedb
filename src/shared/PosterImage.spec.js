import React from 'react';
import { shallow } from 'enzyme';

import { PosterImage } from './PosterImage';

it('renders an image with a correct path and alt attribute', () => {
    const poster = shallow(<PosterImage path="PATH" title="TITLE" />);
    expect(poster.getElement()).toMatchSnapshot();
});
