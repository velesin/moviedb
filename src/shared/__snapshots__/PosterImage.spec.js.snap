// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an image with a correct path and alt attribute 1`] = `
<img
  alt="TITLE"
  src="https://image.tmdb.org/t/p/w90_and_h134_bestv2/PATH"
/>
`;
